rdbms/src/main/java/org/spine3/server/storage/jdbc/JdbcStorageFactory.java

```diff
@@ -58,29 +58,34 @@ private JdbcStorageFactory(DataSourceConfig config) {
     }
 
     @Override
+    public boolean isMultitenant() {
+        return false;
```

Comment:
See the in-memory implementation. We have to add a constructor to pass this variable in.